.travis.yml

```diff
@@ -11,6 +11,13 @@ before_install:
   # Needed to deploy pull request and releases
   - sudo apt-get -y install python-pip python-dev
   - pip install awscli --upgrade --user
+before_deploy:
+  # zip ./dist folder if tag IS present
+  - if [ $TRAVIS_TAG != "" ]; then
+      pushd dist;
+      zip -r  $OLDPWD/release_$TRAVIS_TAG.zip *;
+      popd;
```

Comment:
Was not familiar with pushd/popd. Is this essentially the same as `cd dist` then `cd -` ?

Reply:
Yep, was recommended where I copied it from